app/src/main/java/com/mashup/mobalmobal/ui/profile/domain/model/ProfileItemMapper.kt

```diff
@@ -0,0 +1,72 @@
+package com.mashup.mobalmobal.ui.profile.domain.model
+
+import com.mashup.mobalmobal.R
+import com.mashup.mobalmobal.ui.profile.data.dto.MyDonateDto
+import com.mashup.mobalmobal.ui.profile.data.dto.MyPostDto
+
+fun MyDonateDto.toProfileItems(headerStringId: Int): List<ProfileItem> {
+    return mutableListOf<ProfileItem>().also {
+        if (donates.isNullOrEmpty()) return emptyList()
+
+        it.add(
+            ProfileItem.Header(
+                titleId = headerStringId
+            )
+        )
+
+        it.addAll(
+            donates.map { donate ->
+                ProfileItem.Donation(
+                    donationId = donate.postId.toString(),
+                    imageUrl = donate.post.postImage ?: "",
+                    title = donate.post.title,
+                    description = donate.post.description ?: "",
+                    goalPrice = donate.post.goalPrice.toDouble(),
+                    donatedPrice = donate.post.currentAmount.toDouble(),
+                    startAt = donate.post.startedAt ?: "",
+                    endAt = donate.post.endAt ?: ""
+                )
+            }
+        )
+    }
+}
+
+fun MyPostDto.toProfileItems(headerStringId: Int): List<ProfileItem> {
+    return mutableListOf<ProfileItem>().also {
+        if (posts.isNullOrEmpty()) return emptyList()
+
+        it.add(
+            ProfileItem.Header(
+                titleId = headerStringId
+            )
+        )
+
+        it.addAll(
+            posts.map { post ->
+                ProfileItem.Donation(
+                    donationId = post.postId.toString(),
+                    imageUrl = post.postImage ?: "",
+                    title = post.title,
+                    description = post.description ?: "",
+                    goalPrice = post.goalPrice.toDouble(),
+                    donatedPrice = post.goalPrice.toDouble(),
+                    startAt = post.startedAt ?: "",
+                    endAt = post.endAt ?: ""
+                )
+            }
+        )
+    }
+}
+
+fun Triple<Int, Int, Int>.toSummaryProfileItem() =
```

Comment:
```suggestion
fun createProfileHeaderItems(requestCount, donatedCount, closedCount) =
```
이런 방식도 괜찮아 보여요. Triple로 들어오는 인자들이 어떤 인자들인지 잘 모르겠어요.
그리고 굳이 확장함수가 아니라 ProfileItem 안에 함수를 만들어 줘도 좋을 것 같아요. 
이렇게 확장함수로 작성 되어있으면 작성한 사람만 사용할 수 있을 것 같아요.
왠만하면 namespace는 포함시켜주는게 좋아보입니다.

Reply:
좋은 지적 감사합니다~!